src/modules/QtQuick.Controls/ComboBox.js

```diff
@@ -6,6 +6,8 @@ QmlWeb.registerQmlType({
   properties: {
     count: "int",
     currentIndex: "int",
+    size: { type: "int", initialValue: 1 },
+    // size is non-standard property to qtquick, but useful in dom
```

Comment:
Probably should go somewhere else, but not into the `QtQuick.Controls` module.
E.g. in `QmlWeb/ComboBox` or `QmlWeb.Addons/ComboBox`.

I would prefer if we kept compatibility both ways as much as possible for the modules that are defined in Qt.